pkg/constants/constants.go

```diff
@@ -75,6 +75,10 @@ var (
 	InferenceServiceDefaultingWebhookName       = strings.Join([]string{InferenceServiceName, WebhookServerName, "defaulter"}, ".")
 	PodMutatorWebhookName                       = strings.Join([]string{InferenceServiceName, WebhookServerName, "pod-mutator"}, ".")
 	WebhookFailurePolicy                        = v1beta1.Fail
+	EnableKFServingMutatingWebhook              = "enabled"
+	MatchWorkingNamespaces                      = "MatchWorkingNamespaces"
```

Comment:
what do you think rename this to `ENABLE_WEBOOK_NAMESPACE_SELECTOR` ?

Reply:
ok for me

Reply:
oops my typo it should be `ENABLE_WEBHOOK_NAMESPACE_SELECTOR`